stdlib/public/core/Macros.swift

```diff
@@ -24,4 +23,40 @@
 public macro externalMacro<T>(module: String, type: String) -> T =
     Builtin.ExternalMacro
```

Comment:
Trivial nit: Inconsistent indentation with the rest

```suggestion
  Builtin.ExternalMacro
```